internal/api/api_test.go

```diff
@@ -32,6 +33,10 @@ func setupAPIForTestWithCallback(cb func(*conf.GlobalConfiguration, *storage.Con
 		return nil, nil, err
 	}
 
+	if err := observability.ConfigureLogging(&config.Logging); err != nil {
+		return nil, nil, err
+	}
+
```

Comment:
not worth splitting into a separate PR but i found it easier to read the test logs when they conform to JSON rather than using the standard formatter